### PR TITLE
Add script to verify 1006 issue

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check #[1006](https://github.com/smartdevicelink/sdl_core/issues/1006)

This PR is **ready** for review.

### Summary
A test script has been added to verify that the problem has been fixed.

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1006
+-- Description: PoliciesManager must allow all requested params in case "parameters" field is omitted
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: Please update the description to actual one

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- In case:
+-- 1) Send any single UI-related RPC , UI interface is not supported by the system
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]:
```suggestion
-- 1) Any single UI-related RPC is requested , UI interface is not supported by the system
```

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+-- 2) SDL receives UI.IsReady (available=false) from HMI
+-- Expected result:
+-- 1) SDL must respond "UNSUPPORTED_RESOURCE, success=false, info: UI is not supported by system" to mobile app
+-- Actual result:
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: Please remove the actual result

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+local allParams = {
+  requestParams = requestParams,
+  responseVrParams = responseVrParams
+}
+--[[ Local Functions ]]
+local function sendAddCommand(params)
+  local cid = common.getMobileSession():SendRPC("AddCommand", params.requestParams)
+  common.getHMIConnection():ExpectRequest("VR.AddCommand", params.responseVrParams)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: Please add the expectation of the UI.AddCommand with Times(0)

Comment (HSavynetska) [reply test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: @GetmanetsIrina You can find the updates in the commit ac51975

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua (hunk 3 above)
Comment (HSavynetska) [reply test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: @GetmanetsIrina You can find the updates in the commit ac51975

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua (hunk 2 above)
Comment (HSavynetska) [reply test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: @GetmanetsIrina You can find the updates in the commit ac51975

File: test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua (hunk 1 above)
Comment (HSavynetska) [reply test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua]: @GetmanetsIrina You can find the updates in the commit ac51975

**Comment by dboltovskyi** [comment]
The internal review iteration has been moved to the LuxoftSDL.

**Pull request closed without merging**